dependencies/xenial/unicode-display_width/copyright

```diff
@@ -0,0 +1,32 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: unicode-display_width
+Source: http://github.com/janlelis/unicode-display_width
+
+Files: *
+Copyright: 2011, 2015-2016 Jan Lelis <mail@janlelis.de>
+License: Expat
+
+Files: debian/*
+Copyright: 2016 Foreman developers <foreman-dev@googlegroups.com>
+License: Expat
+Comment: the Debian packaging is licensed under the same terms as the original package.
+
+License: Expat
+ Permission is hereby granted, free of charge, to any person obtaining
+ a copy of this software and associated documentation files (the
+ "Software"), to deal in the Software without restriction, including
+ without limitation the rights to use, copy, modify, merge, publish,
+ distribute, sublicense, and/or sell copies of the Software, and to
+ permit persons to whom the Software is furnished to do so, subject to
+ the following conditions:
+ .
+ The above copyright notice and this permission notice shall be
+ included in all copies or substantial portions of the Software.
+ .
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ EXPRESS OR IMPLIED, INCLUDING BUT NOT LIExpatED TO THE WARRANTIES OF
```

Comment:
Copied this over from another package... which needs to get fixed now, too. Thanks!